v2.2/recommended-production-settings.md

```diff
@@ -39,54 +41,111 @@ Term | Definition
 For added context about CockroachDB's fault tolerance and automated repair capabilities, see [this training](training/fault-tolerance-and-automated-repair.html).
 {{site.data.alerts.end}}
 
+
+## Operating Systems
+
+For information about operating system requirements, see the relevant Install page:
+
+- [Install CockroachDB on Mac](install-cockroachdb-mac.html)
+- [Install CockroachDB on Linux](install-cockroachdb-linux.html)
+- [Install CockroachDB on Windows](install-cockroachdb-windows.html)
+
 ## Hardware
 
 ### Basic hardware recommendations
 
-- Nodes should have sufficient CPU, RAM, network, and storage capacity to handle your workload. It's important to test and tune your hardware setup before deploying to production.
+Nodes should have sufficient CPU, RAM, network, and storage capacity to handle your workload. It's important to test and tune your hardware setup before deploying to production.
+
+#### CPU, memory, and storage recommendations
+
+- At a bare minimum, each node should have **2 GB of RAM and 2 CPUs**.
+
+    More data, complex workloads, higher concurrency, and faster performance require additional resources; as a general rule of thumb, increase the number of CPUs and additional memory to match the requirements of the workload.
 
-- At a bare minimum, each node should have **2 GB of RAM and one entire core**. More data, complex workloads, higher concurrency, and faster performance require additional resources.
     {{site.data.alerts.callout_danger}}
-    Avoid "burstable" or "shared-core" virtual machines that limit the load on a single core.
+    Avoid "burstable" or "shared-core" virtual machines that limit the load on CPU resources.
     {{site.data.alerts.end}}
 
-- For best performance:
-    - Use SSDs over HDDs.
-    - Use larger/more powerful nodes. Adding more CPU is usually more beneficial than adding more RAM.
+- The recommended Linux filesystem is [ext4](https://ext4.wiki.kernel.org/index.php/Main_Page).
+- Avoid using shared storage such as NFS, CIFS and CEPH storage.
+- The optimal configuration for striping more than one device is [RAID 10](https://en.wikipedia.org/wiki/Nested_RAID_levels#RAID_10_(RAID_1+0)). RAID 0 and 1 are also acceptable from a performance perspective.
+
+#### Performance recommendations
+
+For best performance:
+
+- For storage, SSD or NVMe devices will yield the best performance results. The usual volume size should be 300-500 GB.
+
+    Monitor IOPS for higher service times, if they exceed 1-5 ms, you will need to add more devices or expand the cluster to reduce the disk latency. To monitor IOPS, use tools such as `iostat` (part of `sysstat`).
```

Comment:
nit: "Monitor IOPS for higher service times, if they exceed..." > "Monitor IOPS for higher service times. If the IOPS exceed ..."